modules/api/src/main/java/org/eclipse/xpanse/api/OrchestratorApi.java

```diff
@@ -61,6 +63,24 @@ public Response register(@Valid @RequestBody Ocl ocl) {
         return Response.successResponse(successMsg);
     }
 
+    /**
+     * Register new managed service.
+     *
+     * @param ocl object of managed service.
+     * @return response
+     */
+    @PostMapping(value = "/v2/register",
```

Comment:
@iskey As discussed, we can remove the V2 for register and just update the existing register method to use the OclV2

Reply:
> @iskey As discussed, we can remove the V2 for register and just update the existing register method to use the OclV2

Hi @swaroopar , I fix this with some API refactors. just like [this](https://github.com/huaweicloud/xpanse/pull/153#:~:text=POST%3A%20/register%20%0A%20%20%20%20%3D%3D%3E%20%20POST%3A%20%20%20/,GET%3A%20%20%20%20/register/%7BmanagedServiceName%7D)